.github/renovate.json

```diff
@@ -46,6 +46,12 @@
       "matchUpdateTypes": ["minor", "major"],
       "enabled": false
     },
+    {
+      "matchManagers": ["maven"],
+      "matchPackagePrefixes": ["com.graphql-java"],
+      "matchUpdateTypes": ["major"],
```

Comment:
@cmur2 I guess disabling the major updates only, should fix the problem, isn't?

Reply:
It does not say in https://github.com/camunda/zeebe/pull/17785 anything about the upgrade type so I cannot answer that for sure, probably yes.

Reply:
we can adapt it later if needed